pages/index.tsx

```diff
@@ -163,10 +172,14 @@ const ButtonWrapper = styled.div`
   }
 `;
 
-const IssuesPage = () => {
+const IndexPage = ({ data }: { data: TLandingPage }) => {
   const [isMobileMenuOpen, setIsMobileMenuOpen] = useState(false);
   const modalWrapperRef = useRef<HTMLDivElement>(null);
 
+  const hero = data.sections.find(
```

Comment:
A more flexible alternative would be to render all sections something like this:

```
<Main>
{
  data.sections.map(section => {
    // alternatively use a switch statement
    if (section.type === SectionType.hero) {
      return <HeroSection data={section} />
    }
  })
}
</Main>
```

You could also create a separate `Section` component that takes care of this. This component could also import the specific section components dynamically so the bundles are only loaded when necessary